modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -928,5 +948,135 @@ PolygonConcentricCircleMeshGeneratorBase::generate()
   if (flat_side_up)
     MeshTools::Modification::rotate(*mesh0, 180.0 / (Real)_num_sides, 0.0, 0.0);
   mesh0->set_isnt_prepared();
+
+  if (_ring_block_ids.size() > 2 && getParam<bool>("replace_inner_ring_with_delaunay_mesh"))
+  {
+    // remove elements of the inner ring and create an inner-ring external boundary side set
+    std::set<Elem *> deleteable_elems;
+    for (auto & elem : mesh0->element_ptr_range())
+      if (elem->vertex_average().norm() < _ring_radii[0])
```

Comment:
We should know the block_id of the inner ring at this point. So `active_subdomain_elements_ptr_range()` can be used.

Also, `_ring_radii` might not be safe to use when volume preserve is used.

Reply:
Would there possibly two block ids for the elements in the ring?